_data/projects/Animations.yml

```diff
@@ -0,0 +1,12 @@
+name: Animations
+desc: An open source collection of animations, from loaders, to menus, and scroll transitions, and anything in between
+site: https://animations.gq
```

Comment:
@UsmanAhmadSaeed Please change the link to the site or let us know when it is up. putting on hold to merge for the working site link

Reply:
It's up. Thanks!
Currently not a site, just presenting the markdown file, but issue is open for a simple site development.

Reply:
@UsmanAhmadSaeed https://animations.gq/ is not working. not presenting a markdown file